### DISCUSSION TITLE
Implement /cmi sell [item name]

### DISCUSSION THREAD

**Discussion by GiraffeCubed** [Zrips/CMI#2333] [Feature Requests]
**Description of issue or feature request:**
Just like in Essentials, I'd like to suggest that the ability to sell an item by its name or alias be implemented. I.e. /sell dirt, /sell stick, /sell oakwood, /sell birch_door. 

I just recently switched to CMI from EssentialsX and since the ability to /sell [Itemname] is not present, this has broken what is possibly the prime feature of my servers highest rank - autosell. Everyone on my server uses autosell and they're very fond of it. My autosell works by having the player automatically execute /sell [Itemname] when ever they pick up items, so this feature request is pretty vital for us.

Aside from my own servers autosell, it has always been extremely handy to be able to just type /sell dirt or /sell stone and have all of that item be sold from your inventory.

I really think everyone could benefit from this. I also think this could be a great addition to /worth

**Comment by Momshroom** [comment]
I would also much appreciate this.  My players still miss it from Essentials.

**Comment by GiraffeCubed** [comment]
Any more opinions on this? It's really unfortunate that it's missing in CMI.
